lists2safebrowsing.py

```diff
@@ -35,10 +35,14 @@
     ENTITYLIST_SECTIONS,
 )
 from publish2cloud import (
-    publish_to_cloud
+    publish_to_cloud,
+    check_upload_config,
```

Comment:
I might be wrong here. Do we need to import `check_upload_config()` here?

Reply:
Oh yeah, this was left over from when the function was in this file, thanks!